<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="CanvasWidget" name="canvasWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>784</width>
         <height>513</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openSceneFileAction"/>
    <addaction name="openRenderFileAction"/>
    <addaction name="openCommandsFileAction"/>
    <addaction name="saveRenderSettingAction"/>
    <addaction name="saveImageAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="renderSettingDiolagAction"/>
    <addaction name="showCoordSystemAction"/>
    <addaction name="initStateAction"/>
    <addaction name="chooseViewAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="openHelpInfoAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Отрисовка</string>
    </property>
    <addaction name="renderAction"/>
    <addaction name="spectrumRenderAction"/>
    <addaction name="separator"/>
    <addaction name="polarizatiomRenderAction"/>
    <addaction name="dispersionRenderSurfaceAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Дизайн</string>
    </property>
    <addaction name="polarizationTransferAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>226</red>
         <green>228</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>162</red>
         <green>168</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>49</red>
         <green>54</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>65</red>
         <green>72</green>
         <blue>170</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>176</red>
         <green>181</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>226</red>
         <green>228</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>162</red>
         <green>168</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>49</red>
         <green>54</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>65</red>
         <green>72</green>
         <blue>170</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>176</red>
         <green>181</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>49</red>
         <green>54</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>226</red>
         <green>228</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>162</red>
         <green>168</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>49</red>
         <green>54</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>65</red>
         <green>72</green>
         <blue>170</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>49</red>
         <green>54</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>49</red>
         <green>54</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>98</red>
         <green>108</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openSceneFileAction"/>
   <addaction name="openCommandsFileAction"/>
   <addaction name="renderSettingDiolagAction"/>
   <addaction name="saveImageAction"/>
   <addaction name="polarizationTransferAction"/>
   <addaction name="showCoordSystemAction"/>
   <addaction name="renderAction"/>
   <addaction name="spectrumRenderAction"/>
   <addaction name="polarizatiomRenderAction"/>
   <addaction name="dispersionRenderSurfaceAction"/>
   <addaction name="quitAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="openRenderFileAction">
   <property name="text">
    <string>Открыть настройки отрисовки</string>
   </property>
   <property name="statusTip">
    <string>Открыть настройки отрисовки, хранящиеся на диске</string>
   </property>
  </action>
  <action name="saveRenderSettingAction">
   <property name="text">
    <string>Сохранить настройки отрисовки</string>
   </property>
   <property name="statusTip">
    <string>Сохранить насройки отрисовки на диск</string>
   </property>
  </action>
  <action name="saveImageAction">
   <property name="text">
    <string>Сохранить изображение</string>
   </property>
   <property name="statusTip">
    <string>сохранить на диск изображене из рабочей области</string>
   </property>
  </action>
  <action name="renderSettingDiolagAction">
   <property name="text">
    <string>Настройки отрисовки</string>
   </property>
   <property name="statusTip">
    <string>открыть диалог настроек отрисовки отрисовки(Rendering)</string>
   </property>
  </action>
  <action name="initStateAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Начальное положение</string>
   </property>
   <property name="statusTip">
    <string>Выставить сцену в начальное положение</string>
   </property>
  </action>
  <action name="renderAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
   <property name="statusTip">
    <string>RGB рендеринг с традиционными источниками</string>
   </property>
  </action>
  <action name="chooseViewAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Режим выбора ракурса</string>
   </property>
   <property name="statusTip">
    <string>Выбор ракурса(вращение самой сцены)</string>
   </property>
  </action>
  <action name="openHelpInfoAction">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="statusTip">
    <string>Открыть информацию о программе</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="statusTip">
    <string>Выйти из программы</string>
   </property>
  </action>
  <action name="openSceneFileAction">
   <property name="text">
    <string>Открыть файл сцены</string>
   </property>
   <property name="statusTip">
    <string>Открыть файл сцены</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="showCoordSystemAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Показать систему координат</string>
   </property>
   <property name="statusTip">
    <string>Показать мировую систему координат</string>
   </property>
  </action>
  <action name="spectrumRenderAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Спектральный</string>
   </property>
   <property name="statusTip">
    <string>Спектральный рендеринг с традиционнымим источниками</string>
   </property>
  </action>
  <action name="polarizatiomRenderAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C поляризацией</string>
   </property>
   <property name="statusTip">
    <string>Рендеринг с поляризацией с плоским источником</string>
   </property>
  </action>
  <action name="dispersionRenderSurfaceAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Дисперсия</string>
   </property>
   <property name="statusTip">
    <string>Спектральный рендеринг с плоским источником(дисперсия)</string>
   </property>
  </action>
  <action name="openCommandsFileAction">
   <property name="text">
    <string>Открыть командный файл</string>
   </property>
   <property name="statusTip">
    <string>Командный режим</string>
   </property>
  </action>
  <action name="polarizationTransferAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Перенос поляризации</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CanvasWidget</class>
   <extends>QWidget</extends>
   <header>CanvasWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
